python/examples/cache/unconstrained_cache.py

```diff
@@ -0,0 +1,22 @@
+import asyncio
+import sys
+import traceback
+
+from beeai_framework.cache.unconstrained_cache import UnconstrainedCache
+from beeai_framework.errors import FrameworkError
+
+cache: UnconstrainedCache[int] = UnconstrainedCache()
+
+
+async def main() -> None:
+    await cache.set("a", 1)
+    print(await cache.has("a"))  # True
+    print(await cache.size())  # 1
```

Comment:
Is there a reason you didn't include the entire example from TS? This is only a small part of the original